quartz/plugins/transformers/description.ts

```diff
@@ -5,12 +5,19 @@ import { escapeHTML } from "../../util/escape"
 
 export interface Options {
   descriptionLength: number
+  replaceExternalLinks: boolean
 }
 
 const defaultOptions: Options = {
   descriptionLength: 150,
+  replaceExternalLinks: true,
```

Comment:
do we still need this option? imo because we split on `/\.\s/` now, urls should be fine?

Reply:
Well, I guess we don't _need_ it. It doesn't split links halfway through anymore, even if the option is set to `false`.

Currently it replaces external links by their domain. This means that very long external links don't use most if not all of the description length.

I would suggest keeping the option in, because in my opinion external links should generally not be part of the description of a page, as they are inherently seperate from the page. You know, begin an external page and all.

That being said, another option could be simply defaulting the value to `false`.

Reply:
imo `https://domain.tld/ -> domain` doesn't tell me it's a url still, we should rather just strip the protocol field along with any query params and anchors but keep the host + path

Reply:
That's a fair point. I have updated the regex.

Parsing a description with `https://domain.tld/some_page/another_page?query=hello&target=world` will now result in `domain.tld/some_page/another_page`. I have also updated the documentation to reflect that.

Currently, the `replaceExternalLinks` option is still default `true`. Do you want to default it to `false` or keep it as `true`?

Reply:
nah, this is fine, thank you!